### PR TITLE
InstructorFeedbackEditPage: add documentation for selectConstSumPointsOptions method #7910

### LINKED ISSUE #7910: InstructorFeedbackEditPage: add documentation for selectConstSumPointsOptions method
The `selectConstSumPointsOptions()` method in `InstructorFeedbackEditPage` is used to switch between the points distribution scheme for const sum question. Though the method is understandable, the arguments to pass to the method is still a mystery even after looking at the code -

```java
public void selectConstSumPointsOptions(String pointsOption, int questionNumber) {
    markRadioButtonAsChecked(
           browser.driver.findElement(By.id("constSumPoints" + pointsOption + "-" + questionNumber)));
}
```

One would have to check the templates or inspect HTML to find out the ID of the radio buttons and deduce the value for the String parameter. It would be very helpful to devs if we added a small documentation string above this method -

```java
/**
 * Selects the points distribution scheme for a const sum question.
 * 
 * @param pointsOption Value of "Total" will select total number of points for all options points distribution scheme,
 *                     or "PerOption" will select number of points per option points distribution scheme.
 * @param questionNumber question number of the const sum question.
 */
```

### PR DESCRIPTION
**Pull request by Shels1909**
Fixes #7910 

Hi, This is my first PR, and I am really hoping to contribute more! I added the documentation that VamsiSangam had put in the open issue. I ran the both of the static tests and they seemed to have passed. If there is anything I need to change please let me know.

### CODE REVIEW

**Comment by whipermr5** [comment]
Hi @Shels1909, welcome to TEAMMATES! Do follow the [guidelines](https://github.com/TEAMMATES/teammates/blob/master/docs/process.md#step-4-submit-a-pr) exactly - the PR title is still wrong.

**Comment by Shels1909** [comment]
Hi @whipermr5 thanks! I changed the PR title to reflect the relevant issue name.

File: src/test/java/teammates/test/pageobjects/InstructorFeedbackEditPage.java [hunk 1]
```diff
@@ -1005,6 +1005,14 @@ public void selectMsqGenerateOptionsFor(String generateFor, int questionNumber)
                 generateFor);
     }
 
+    /**
+     * Selects the points distribution scheme for a const sum question.
+     *
+     * @param pointsOption Value of "Total" will select total number of points for all options points distribution scheme,
```
Comment (VamsiSangam) [inline src/test/java/teammates/test/pageobjects/InstructorFeedbackEditPage.java]: Captalise `T` in `total number of points for all options`, and `N` in `number of points per option`. Also wrap them in `<i></i>`. Like this -

> Value of "Total" will select `<i>`Total number of points for all options`</i>` points distribution scheme, or "PerOption" will select `<i>`Number of points per option`</i>` points distribution scheme.

File: src/test/java/teammates/test/pageobjects/InstructorFeedbackEditPage.java [hunk 2]
```diff
@@ -1005,6 +1005,14 @@ public void selectMsqGenerateOptionsFor(String generateFor, int questionNumber)
...
+     *                     or "PerOption" will select number of points per option points distribution scheme.
+     * @param questionNumber question number of the const sum question.
+     */
+
```
Comment (VamsiSangam) [inline src/test/java/teammates/test/pageobjects/InstructorFeedbackEditPage.java]: This new line can be removed.

**Comment by VamsiSangam** [comment]
@whipermr5 this PR is ready to be reviewed by a senior dev.

**Pull request merged**